app/src/main/java/chat/rocket/android/helper/OkHttpHelper.java

```diff
@@ -48,4 +63,28 @@ public static OkHttpClient getClientForWebSocket() {
     }
     return httpClientForWS;
   }
+
+  public static String getContentType(String uri) {
+    Request request = new Request.Builder()
+        .url(uri)
+        .head()
+        .build();
+    getDefaultHttpClient().newCall(request).enqueue(new Callback() {
+      @Override
+      public void onFailure(@NonNull Call call, @NonNull IOException ioException) {
+        ioException.printStackTrace();
+      }
+
+      @Override
+      public void onResponse(@NonNull Call call, @NonNull final Response response)
+          throws IOException {
+        if (!response.isSuccessful()) {
+          throw new IOException("Unexpected code: " + response);
+        } else {
+          contentType = response.header("Content-Type");
+        }
+      }
+    });
+    return contentType;
```

Comment:
To get the `contentType` we need to do a network call - blocking operation, so we need to do an async task, right? There are the following possibilities: 

1) Get the `contentType` using the `HttpURLConnection ` with the `AsyncTask`.
2) Get the `contentType` using the `HttpURLConnection ` with `RxJava`.
3) Get the `contentType` using the `OkHttp ` which is currently used on RC app - `OkHttp` is a built-in response cache, a better cookie model, a better headers model and a better call model.

So, how we should handle this requirement?